src/components/routes/Host/HostListingCard/HostListingCard.tsx

```diff
@@ -41,20 +42,20 @@ const HostListingCard = (props: Props): JSX.Element => {
     <HostListingCardContainer className="host-listing-card">
       <div className="host-listing-meta">
         <h1>{title}</h1>
-        {(city || state || country) && <h2>{formatAddress(city, state, country)}</h2>}
+        <h2>{(city || state || country) ? formatAddress(city, state, country) : ''}</h2>
         <div className="bee-flex-div" />
         <h3>Last edited: {format(updatedAt, 'MM/DD/YY [at] hh:mmA')}</h3>
+        <h4>
```

Comment:
i think you can do a button with the `clear` and `suffix` properties to achieve something similar

Reply:
Thanks, I'll look at that